File: code/js/src/views/user/TrainerViews/models/exercises/Exercise.ts
class Exercise{
    id: number;
    name: string;
    muscleGroup: string[];
    modality: string;
    isFavorite: boolean = false;
    constructor(id: number, name: string, muscleGroup: string[], modality: string, isFavorite: boolean = false) {
        this.id = id;
        this.name = name;
        this.muscleGroup = muscleGroup;
        this.modality = modality;
        this.isFavorite = isFavorite;
    }
}

export default Exercise;

File: code/jvm/src/main/kotlin/pt/isel/leic/ptgest/domain/exercise/model/Exercise.kt
package pt.isel.leic.ptgest.domain.exercise.model

import pt.isel.leic.ptgest.domain.workout.Modality
import pt.isel.leic.ptgest.domain.workout.MuscleGroup

data class Exercise(
    val id: Int,
    val name: String,
    val muscleGroup: List<MuscleGroup>,
    val modality: Modality
)
